src/style.css

```
@@ -0,0 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search__bar {
+  display: flex;
+  margin: 20px 8px 0;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__bar--big {
+  height: 70px;
+}
```

```suggestion
  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
  height: 70px;
}
```
same for other elements.